src/helpers/BoxHelper.js

```diff
@@ -47,6 +47,8 @@ BoxHelper.prototype.update = function ( object ) {
 
 		_box.setFromObject( this.object );
 
+		this.geometry.boundingBox = null;
```

Comment:
I'm not sure this change is correct. The idea of using `setFromObject()` is to avoid the computation of `BufferGeometry.boundingBox` over and over again.

Reply:
Hi , recently we update to v111 and having an issue with these code:

...
var sourceBoxHelper = new THREE.BoxHelper(sourceObject)
var sourceBox = new THREE.Box3();
...
sourceBoxHelper.update()
sourceBox.setFromObject(sourceBoxHelper)
...

Here the sourceBox will not get the correct updated geometry information (which works fine before v111).

I found the problem is that the  latest optimize for .expandByObject() #17940 which will always set the boundingbox value when the BoxHelper first constructed. 

Then afterwards when BoxHelper update() get called again, the boundingbox will not be set to the updated value (the boundingbox will not be calculated when update() get called, I think this is not a correct behavior).